tests/lib/rules/jsx-wrap-multilines.js

```diff
@@ -147,96 +216,258 @@ ruleTester.run('jsx-wrap-multilines', rule, {
     }, {
       code: RETURN_PAREN
     }, {
+      code: RETURN_SINGLE_LINE,
+      options: [{return: true}]
+    }, {
+      code: RETURN_PAREN,
+      options: [{return: true}]
+    }, {
+      code: RETURN_NO_PAREN,
+      options: [{return: 'ignore'}]
+    }, {
       code: RETURN_NO_PAREN,
       options: [{return: false}]
     }, {
       code: DECLARATION_TERNARY_SINGLE_LINE
     }, {
       code: DECLARATION_TERNARY_PAREN
     }, {
+      code: DECLARATION_TERNARY_SINGLE_LINE,
+      options: [{declaration: true}]
+    }, {
+      code: DECLARATION_TERNARY_PAREN,
+      options: [{declaration: true}]
+    }, {
+      code: DECLARATION_TERNARY_NO_PAREN,
+      options: [{declaration: 'ignore'}]
+    }, {
       code: DECLARATION_TERNARY_NO_PAREN,
       options: [{declaration: false}]
     }, {
       code: ASSIGNMENT_TERNARY_SINGLE_LINE
     }, {
       code: ASSIGNMENT_TERNARY_PAREN
     }, {
+      code: ASSIGNMENT_TERNARY_SINGLE_LINE,
+      options: [{assignment: true}]
+    }, {
+      code: ASSIGNMENT_TERNARY_PAREN,
+      options: [{assignment: true}]
+    }, {
+      code: ASSIGNMENT_TERNARY_NO_PAREN,
+      options: [{assignment: 'ignore'}]
+    }, {
       code: ASSIGNMENT_TERNARY_NO_PAREN,
       options: [{assignment: false}]
     }, {
       code: DECLARATION_SINGLE_LINE
     }, {
       code: DECLARATION_PAREN
     }, {
+      code: DECLARATION_SINGLE_LINE,
+      options: [{declaration: true}]
+    }, {
+      code: DECLARATION_PAREN,
+      options: [{declaration: true}]
+    }, {
+      code: DECLARATION_NO_PAREN,
+      options: [{declaration: 'ignore'}]
+    }, {
       code: DECLARATION_NO_PAREN,
       options: [{declaration: false}]
     }, {
       code: ASSIGNMENT_SINGLE_LINE,
+      options: [{declaration: 'ignore'}]
+    }, {
+      code: ASSIGNMENT_SINGLE_LINE,
       options: [{declaration: false}]
     }, {
       code: ASSIGNMENT_PAREN
     }, {
+      code: ASSIGNMENT_PAREN,
+      options: [{assignment: true}]
+    }, {
+      code: ASSIGNMENT_NO_PAREN,
+      options: [{assignment: 'ignore'}]
+    }, {
       code: ASSIGNMENT_NO_PAREN,
       options: [{assignment: false}]
     }, {
+      code: ARROW_PAREN
+    }, {
+      code: ARROW_SINGLE_LINE
+    }, {
       code: ARROW_PAREN,
-      options: []
```

Comment:
should we not keep this test against "the default"?

Reply:
The two tests above these are using the defaults.